src/robusta/cli/main.py

```diff
@@ -87,11 +87,16 @@ def set_pod_configs_for_small_clusters(self):
         )
 
 
-def guess_cluster_name():
+def guess_cluster_name(context):
     with click_spinner.spinner():
         try:
             all_contexts, current_context = config.list_kube_config_contexts()
+            if context is not None:
+                for i in range(len(all_contexts)):
```

Comment:
if there is no context with the context name given to the function
i would have typer print something like, no context exists with the name '{context}', your current context is ... 
so that way the user has an idea why the context he inserted didnt display the cluster he wanted